src/components/switch.re

```diff
@@ -1,8 +1,18 @@
 [@bs.module "react-native"] external view: ReasonReact.reactClass = "Switch";
 
+let makeTrackColor = (trackColor: option(array((bool, string)))) => {
+  let arr = switch (trackColor) {
+  | None => [||]
+  | Some(x) => x
+  };
+  let f = (s: option(string)): Js.nullable(string) => {if (s == None) Js.Nullable.null else Js.Nullable.fromOption(s)};
+  Js.Dict.fromArray(Array.map(y => (Js_string.make(y), f(UtilsRN.option_map(snd, Js.Array.find((x) => {fst(x) == y}, arr)))),  [|true, false|]))
```

Comment:
Could you run refmt on this to make it more easy to read? Thanks!

Reply:
I ran refmt, but also made a few trivial changes.

• replaced `UtilsRN.option_map` with `Belt.Option.map`. As the former is simply another implementation of the latter with reversed order of arguments, perhaps it should be replaced throughout the repo. However, I can undo that change if it is preferred to avoid using `Belt`.
• added the prop to documentation

I also force pushed the commit to reduce the number of commits based on feedback from my first accepted PR into this repo.